lib/datasets_test.go

```diff
@@ -384,103 +385,118 @@ func TestDatasetRequestsGet(t *testing.T) {
 		t.Fatalf("error loading dataset: %s", err.Error())
 	}
 
+	moviesDs.OpenBodyFile(node.Repo.Filesystem())
+	moviesBody := make([]interface{}, 0)
+	moviesBodyFile := moviesDs.BodyFile()
+	reader := dsio.NewCSVReader(moviesDs.Structure, moviesBodyFile)
+	for {
+		ent, err := reader.ReadEntry()
+		if err != nil {
+			if err.Error() == "EOF" {
+				break
+			}
+			t.Fatal(err.Error())
+		}
+		moviesBody = append(moviesBody, ent.Value)
+	}
+
 	cases := []struct {
```

Comment:
muuuuuuuch better. in a later PR we should add tests for the only missing branches of the `Selector ==` if statement: `transform.script` and `viz.script`. Thankfully these tests are doing something meaningful :)